apps/notification-producer/src/producers/trade/getTradeNotifications.ts

```diff
@@ -1,32 +1,32 @@
-import {
-  COW_PROTOCOL_SETTLEMENT_CONTRACT_ADDRESS,
-  SupportedChainId,
-} from '@cowprotocol/cow-sdk';
+import { COW_PROTOCOL_SETTLEMENT_CONTRACT_ADDRESS, ETH_FLOW_ADDRESSES, SupportedChainId } from '@cowprotocol/cow-sdk';
 import { PushNotification } from '@cowprotocol/notifications';
-import { Erc20Repository, getViemClients } from '@cowprotocol/repositories';
+import { Erc20Repository, getViemClients, OnChainPlacedOrdersRepository } from '@cowprotocol/repositories';
 import { getAddress, parseAbi } from 'viem';
 import { bigIntReplacer, logger } from '@cowprotocol/shared';
 import { fromTradeToNotification } from './fromTradeToNotification';
 import { fromOrderInvalidationToNotification } from './fromOrderInvalidationToNotification';
 
 const EVENTS = parseAbi([
   'event OrderInvalidated(address indexed owner, bytes orderUid)',
-  'event Trade(address indexed owner, address sellToken, address buyToken, uint256 sellAmount, uint256 buyAmount, uint256 feeAmount, bytes orderUid)',
+  'event Trade(address indexed owner, address sellToken, address buyToken, uint256 sellAmount, uint256 buyAmount, uint256 feeAmount, bytes orderUid)'
 ]);
 
+const ethFlowAddresses = Object.values(ETH_FLOW_ADDRESSES).map(getAddress);
```

Comment:
Why do you map it to getAddress?

Reply:
To have it checksummed, because values in logs checksummed

Reply:
We use it to check `ethFlowAddresses.includes(owner)`